### ISSUE TITLE
files missing cjrtnc.leaningtech.com ?

### ISSUE THREAD

**Issue by jezzarb94** [leaningtech/cheerpj-meta#6]
it apears im getting a 404 when  i run my app

cjrtnc.leaningtech.com/cheerpj/lib/accessibility.properties Failed to load resource: the server responded with a status of 404 ()
cjrtnc.leaningtech.com/cheerpj/lib/swing.properties Failed to load resource: the server responded with a status of 404 ()
cjrtnc.leaningtech.com/cheerpj/lib/net.properties Failed to load resource: the server responded with a status of 404 ()
cjrtnc.leaningtech.com/cheerpj/lib/fonts/LucidaSansRegular.ttf Failed to load resource: the server responded with a status of 404 ()
2cjrtnc.leaningtech.com/cheerpj/lib/fonts/badfonts.txt Failed to load resource: the server responded with a status of 404 ()
2cjrtnc.leaningtech.com/cheerpj/lib/fonts/fallback Failed to load resource: the server responded with a status of 404 ()

**Comment by alexp-sssup** [comment]
This is not a problem, CheerpJ implements a virtual filesystem over HTTP and it's perfectly valid for some files to be missing.
